locale/zh_CN.po

```diff
@@ -5247,7 +5248,7 @@ msgstr "显示全部"
 msgid ""
 "Display all index items that contain given substring. Search is case "
 "insensitive."
-msgstr "显示包含给定子串的所有索引项. 搜索是大小写无关的。"
+msgstr "显示包含给定子串的所有索引项. 搜索是大小写无关的."
```

Comment:
nit. `。`?

Reply:
There we go again... my delivery on June 25th, 2022 used 。
![Screen Shot 2022-07-06 at 2 02 54 AM](https://user-images.githubusercontent.com/82932766/177388765-95c37486-b646-4585-98f1-6dd0added967.png)